src/utilities/service-utils.test.ts

```diff
@@ -215,6 +215,30 @@ describe("ServiceUtils", () => {
             expect(result.resultObject).toBeUndefined();
         });
 
+        test("when response.data.resultObject is not null, it wraps the result object in a record.", () => {
```

Comment:
While this test works, I wonder if we would get more bang for our buck by converting the seeded `resultObject` to a plain ol' JS object (the way it would actually be returned from the API), and then asserting that the `result.resultObject` is an instance of `StubResourceRecord`.

Reply:
Yeah that makes sense @brandongregoryscott . Admittedly I am a total Jest noob, so I appreciate the feedback. I'll take another crack at it and update the PR.

Reply:
No worries 👍 There _should_ be a `toBeInstanceOf` matcher, just a heads up!

Reply:
Ah shoot! I implemented it without using that helper. I'm refactoring now. Thanks for the heads up.

Reply:
Okay @brandongregoryscott it's ready again 😄 . Let me know if this is better,

Reply:
Perfect! Thanks for updating that.